modules/stochastic_tools/src/utils/BootstrapCalculators.C

```diff
@@ -56,6 +59,25 @@ BootstrapCalculator::BootstrapCalculator(const libMesh::ParallelObject & other,
               "The supplied levels must be less than one");
 }
 
+std::vector<Real>
```

Comment:
This seems unrelated to the Cartesian product sampling.  Is that right?  Did you mean to include it in this PR?  Since it has its own separate issue, that's OK, but I just wanted to check.